### PR TITLE
doc/rgw: s/[Zz]one [Gg]roup/zonegroup/g

### PR DESCRIPTION
**Pull request by zdover23**
Change all instances of "Zone Group" and "Zone group" and "zone group" to "Zonegroup" or "zonegroup".

Signed-off-by: Zac Dover <zac.dover@proton.me>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/radosgw/multisite.rst [hunk 1]
```diff
@@ -16,28 +16,28 @@ Beginning with the Kraken release, Ceph supports several multi-site
...
-  zones. Objects that are stored to zones in one zone group within the same
-  realm as another zone group share a global object namespace, which ensures
-  unique object IDs across zone groups and zones.
+  supports multiple zonegroups, with each zonegroup containing one or more
+  zones. Objects that are stored to zones in one zonegroup within the same
+  realm as another zonegroup share a global object namespace, which ensures
+  unique object IDs across zonegroups and zones.
 
 - **Multiple Realms:** Beginning with the Kraken Ceph release, the Ceph Object
   Gateway supports something called "realms". Realms have a globally unique
```
Comment (anthonyeleven) [inline doc/radosgw/multisite.rst:36]: S/something called//

**Review by anthonyeleven — APPROVED** [review]
Approved to obviate streamlining but would like to see changes

**Comment by zdover23** [comment]
> Approved to avoid handshaking but would like to see changes

These changes are really just scaffolding. They're here to help me understand the pragmatic material that multisite.rst describes. I have made a note to myself to return to this PR and to address all of your concerns. I appreciate the prospective approval of this PR, which makes it possible for me to push ahead with the multisite.rst refinement project.

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/50136 - Quincy backport
https://github.com/ceph/ceph/pull/50137 - Pacific backport

**Pull request merged**